### ISSUE TITLE
Transcription command keeps redirecting to Swiftlink page

### ISSUE THREAD

**Issue by gon-pm** [djmango/obsidian-transcription#37]
Hey there, i was using Transciption with Whisper ASR via Docker. Seemed to work fine for a while until i tried to run the command again and now it redirects to [Swiftink](https://www.swiftink.io/login/?callback=obsidian://swiftink_auth) regardless of the transcription engine used.
For reference, these are the settings that i've used
![imagen](https://github.com/djmango/obsidian-transcription/assets/47288005/eaa9ccb0-d1f5-40b8-b585-1e728b218a77)
And those are the contents of `data.json`
```
{
  "timestamps": true,
  "timestampFormat": "HH:mm:ss",
  "translate": false,
  "language": "es",
  "verbosity": 1,
  "whisperASRUrl": "http://localhost:9000",
  "debug": false,
  "transcriptionEngine": "whisper_asr",
  "embedAdditionalFunctionality": true,
  "embedSummary": true,
  "embedOutline": true,
  "embedKeywords": true,
  "swiftink_access_token": "<redacted>",
  "swiftink_refresh_token": "<redacted>"
}
```
Note that i did sign up to Swiftink, but it doesn't work, it just uploads the file but it doesn't transcribe it.
Edit: Mispelled Swiftink
Thanks!

**Comment by djmango** [comment]
Confirmed. Will ship a fix this week

**Comment by djmango** [comment]
Fixed in 3.2.2
